Source/FluentSpreadsheets.GoogleSheets/Extensions/BorderExtensions.cs

```diff
@@ -7,7 +8,7 @@ internal static class BorderExtensions
 {
     public static Border? ToGoogleSheetsBorder(this BorderStyle borderStyle)
     {
-        string? sheetsBorderStyle = GetGoogleSheetsBorderStyle(borderStyle.Type);
+        var sheetsBorderStyle = GetGoogleSheetsBorderStyle(borderStyle.Type);
```

Comment:
Here it's var but in ClosedXmlHandler it's explicit for some reason

Reply:
It is a hint by Rider, I have a configuration set to "use var when evident", and it thinks that here is evident, and in `ClosedXmlHandler` it is not. I kinda *feel* that way too, but can't really explain why :)